umap/views.py

```diff
@@ -1484,7 +1484,7 @@ def webmanifest(request):
 
 def logout(request):
     do_logout(request)
-    return HttpResponseRedirect("/")
+    return HttpResponseRedirect(getattr(settings, "LOGOUT_REDIRECT_URL", "/"))
```

Comment:
I would be more inclined to define a default value (`"/"`) for `LOGOUT_REDIRECT_URL` [in base.py](https://github.com/umap-project/umap/blob/d147fb114b35faa74e7151c5e3e4bf7f883d4627/umap/settings/base.py#L163) and then use directly the setting here:

```suggestion
    return HttpResponseRedirect(settings.LOGOUT_REDIRECT_URL)
```